30daysOFcode_Swift/day_22.swift

```diff
@@ -0,0 +1,51 @@
+// Start of Node class
+class Node {
+    var data: Int
+    var left: Node?
+    var right: Node?
+
+    init(d : Int) {
+        data  = d
+    }
+} // End of Node class
+
+// Start of Tree class
+class Tree {
+    func insert(root: Node?, data: Int) -> Node? {
+        if root == nil {
+            return Node(d: data)
+        }
+
+        if data <= (root?.data)! {
+            root?.left = insert(root: root?.left, data: data)
+        } else {
+            root?.right = insert(root: root?.right, data: data)
+        }
+
+        return root
+    }
+
+       func getHeight(root: Node?) -> Int {
+        if let currentRoot = root {
+            let leftHeight = getHeight(root: currentRoot.left)
+            let rightHeight = getHeight(root: currentRoot.right)
+            
+            return 1 + max(leftHeight, rightHeight)
+        }
+        
+        return -1
+    }
+

```

Comment:
this is the required function to be coded there and hence i have completed this function portion.